sdk/ml/azure-ai-ml/azure/ai/ml/operations/_component_operations.py

```diff
@@ -69,6 +69,21 @@ class ComponentOperations(_ScopeDependentOperations):
     You should not instantiate this class directly. Instead, you should
     create an MLClient instance that instantiates it for you and
     attaches it as an attribute.
+
+    :param operation_scope: The operation scope.
+    :type operation_scope: ~azure.ai.ml._scope_dependent_operations.OperationScope
+    :param operation_config: The operation configuration.
+    :type operation_config: ~azure.ai.ml._scope_dependent_operations.OperationConfig
+    :param service_client: The service client for API operations.
+    :type service_client: Union[
+                            ~azure.ai.ml._restclient.v2022_10_01.AzureMachineLearningWorkspaces,
```

Comment:
indent too long?

Reply:
Just align the previous line, I will modify this